src/nsm.h

```diff
@@ -45,7 +47,30 @@
 /*     return ERR_OK;                                        */
 /* }                                                         */
 /*                                                           */
-/* static nsm_client_t *nsm = 0                              */
+/* void                                                      */
+/* cb_nsm_show ( void *userdata )                            */
+/* {                                                         */
+/*     do_show_ui();                                         */
+/*     nsm_send_is_shown ( nsm );                            */
+/* }                                                         */
+/*                                                           */
+/* void                                                      */
+/* cb_nsm_hide ( void *userdata )                            */
+/* {                                                         */
+/*     do_hide_ui();                                         */
+/*     nsm_send_is_hidden ( nsm );                           */
+/* }                                                         */
+/*                                                           */
+/* gboolean                                                  */
+/* poll_nsm()                                                */
+/* {                                                         */
+/*     if (nsm)                                              */
+/*     {                                                     */
+/*         nsm_check_nowait(nsm);                            */
```

Comment:
a space between the function name and parenthesis is needed here, so it follows the style of the rest of the file

Reply:
ups, will do.

Reply:
Fixed now.